cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -194,7 +194,7 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             this.$('input.date').datepicker({'dateFormat': 'm/d/yy'});
             this.$('input.time').timepicker({
                 'timeFormat' : 'H:i',
-                'forceRoundTime': true
+                'forceRoundTime': false
```

Comment:
@cahrens you mean TimedExaminationPreferenceEditor (here)[https://github.com/simbs/edx-platform/blob/mariana/remove-force-round-time/cms/static/js/views/modals/course_outline_modals.js#L298] ? That is extends from AbstractEditor, not BaseDateEditor. Either way, wouldn't it call the timepicker with it's default value instead of what is set in BaseDate there? Which means that if the deafult is changed there would be a problem, it's also why I set it explicitly for BaseDate.